test/app/features/appstate/domain/usecases/app_preferences_usecase_test.dart

```diff
@@ -164,10 +200,9 @@ void main() {
       );
 
       expect(value.isRight(), true);
-      expect(
-        value.getOrElse(() => null),
-        AppRoute('/authentication/sign_in_stealth'),
-      );
+      value.forEach((route) {
+        expect(route, AppRoute('/authentication/sign_in_stealth'));
+      });
```

Comment:
```suggestion
      expect(
        value.fold((l) => l, (r) => r),
        AppRoute('/authentication/sign_in_stealth'),
      );
```

Acredito que utilizar o `.fold` facilita a leitura do teste.

Reply:
feito